src/libraries/Microsoft.PowerFx.Core/Public/Values/DateTimeValue.cs

```diff
@@ -28,6 +28,12 @@ public override void Visit(IValueVisitor visitor)
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
+            if (settings.UseCompactRepresentation)
+            {
+                sb.Append($"DateTime({Value.Date.Year},{Value.Date.Month},{Value.Date.Day},{Value.TimeOfDay.Hours},{Value.TimeOfDay.Minutes},{Value.TimeOfDay.Seconds},{Value.TimeOfDay.Milliseconds})");
```

Comment:
>DateTime({Value.Date.Year},{Value.Date.Month},{Value.Date.Day},{Value.TimeOfDay.Hours},{Value.TimeOfDay.Minutes},{Value.TimeOfDay.Seconds},{Value.TimeOfDay.Milliseconds}) [](http://example.com/codeflow?start=28&length=170)

Any reason why we don't use the DateTime function for all scenarios? It is guaranteed to work in all locales.

Reply:
Can you add a comment saying why we use the DateTimeValue function here (instead of DateTime, which would be more compact)?

Reply:
This was a result of some prior conversation on top of this subject. I'm removing the special case and using DateTime only.